src/common/services/api/profile.service.js

```diff
@@ -29,7 +29,7 @@ class Profile{
     return this.cortexApiService.get({
         path: ['profiles', this.cortexApiService.scope, 'default'],
         zoom: {
-          paymentMethods: 'paymentmethods:element[]'
+          paymentMethods: 'paymentmethods:element[],paymentmethods:element:bankaccount,paymentmethods:element:bankaccount:element,paymentmethods:element:creditcard,paymentmethods:element:creditcard:element'
```

Comment:
@OzzieOrca , @adammeyer ,guys, do you think it would be good to make zoom.paymentMethods have dynamic value? In case where I need recurring gifts passed along with payment methods I add more loading time when you don't really need extra data for checkout. Let me know what you guys think. Thanks